backend/src/services/upload-file/upload-file.service.ts

```diff
@@ -0,0 +1,33 @@
+import cloudinary, { UploadApiOptions } from 'cloudinary';
+import DatauriParser from 'datauri/parser';
+import { ENV } from '~/common/enums/enums';
+
+interface UploadFileResponse {
+  url: string;
+  bytes: number;
+}
```

Comment:
```suggestion
type UploadFileResponse = {
  url: string;
  bytes: number;
}
```

move it to `backend/src/common/types/file/upload-file-response.type.ts`